src/RemoteRepository/SilverStripeRemoteRepository.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace SilverStripe\ApiDocs\RemoteRepository;
+
+use Doctum\RemoteRepository\AbstractRemoteRepository;
+
+class SilverStripeRemoteRepository extends AbstractRemoteRepository
+{
+
+    public function getFileUrl($projectVersion, $relativePath, $line)
+    {
+        // $this->localPath is the root path to packages
+        // Example: /mnt/Dev/@code-lts/@doctum-fork/api.silverstripe.org/data/packages/
+        // $this->name is empty, and not used
+        // $relativePath for the current file
+        // Example "silverstripe/admin/code/AdminRootController.php"
+        $pathParts = explode('/', $relativePath, 3);
+        // Example: [silverstripe, admin, code/AdminRootController.php]
+        $this->name = $pathParts[0] . '/' . $pathParts[1];
+        $url = 'https://github.com/' . $this->name . '/blob/' . $this->buildProjectPath($projectVersion, $relativePath);
```

Comment:
```suggestion
        $url = 'https://github.com/' . $this->name . '/blob/' . $this->buildProjectPath($projectVersion, $pathParts[2]);
```

@williamdes just checked this out locally, the generated links aren't quite right, I'm getting:

`https://github.com/silverstripe/framework/blob/4/silverstripe/framework/src/Core/Config/Configurable.php#L20` for example  - note that `silverstripe/framework` is in there twice. Think the above would fix it?

Reply:
Hmm no well that fixes one part - the duplication - but in this case, the repo URL for silverstripe/admin is `silverstripe/silverstripe-admin/` - so using the checked-out path isn't going to work there. I'll see if I can work something out.

Reply:
Alright, this seems to work:

```php
        $packageConfig = Config::getConfig()['packages'][$this->name];
        $rootPath = $packageConfig['repository'];
        if (substr($rootPath, -4) === '.git') {
            $rootPath = substr($rootPath, 0, -4);
        }
        if (isset($packageConfig['versionmap'])) {
            $versionMaps = Config::getConfig()['versionmaps'];
            $projectVersion = $versionMaps[$packageConfig['versionmap']][(string) $projectVersion];
        }
        $url = $rootPath . '/blob/' . $this->buildProjectPath($projectVersion, $pathParts[2]);
```

Takes into account both the correct URL for the repo, the correct version to use to align everything, and removing the stray `.git` extension on the repo